sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.query.sql

```diff
@@ -44,7 +44,38 @@ WITH _current AS (
     CAST(browser_engagement_uri_count > 0 AS INT64) &
     CAST(browser_engagement_active_ticks > 0 AS INT64) AS days_desktop_active_bits,
     {% endif %}
-    * EXCEPT(submission_date)
+    client_id,
+    sample_id,
+    first_run_date,
+    durations,
+    days_seen_session_start_bits,
+    days_seen_session_end_bits,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    android_sdk_version,
+    locale,
+    city,
+    country,
+    isp,
+    app_build,
+    app_channel,
+    app_display_version,
+    architecture,
+    device_manufacturer,
+    device_model,
+    telemetry_sdk_build,
+    first_seen_date,
+    is_new_profile,
```

Comment:
It'd be good if this query's column order matched the [`schema.yaml` column order](https://github.com/mozilla/bigquery-etl/blob/a46e060952c06bfa209ebac81d34bced1ac15d76/sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.schema.yaml#L42-L62) (also in the `_previous` CTE):
```suggestion
    app_build,
    app_channel,
    app_display_version,
    architecture,
    device_manufacturer,
    device_model,
    telemetry_sdk_build,
    first_seen_date,
    is_new_profile,
    isp,
```
Or alternatively, the `schema.yaml` could be updated to match this column order.